test/dual_tests.jl

```diff
@@ -1 +1,218 @@
+@testset "Duals: Test Basic" begin
+    m_slave = Model(() -> POI.Optimizer(GLPK.Optimizer()))
```

Comment:
why the model name `m_slave`? Overall this is terminology that the programming community is moving away from. If you want to convey the corresponding idea, `submodel` or `m_inner` might be better

Reply:
I got the tests from https://github.com/JuliaStochOpt/ParameterJuMP.jl/blob/master/test/tests.jl#L22, but I agree with you, the name `model` is much better.